addons/sourcemod/translations/gokz-core.phrases.txt

```diff
@@ -311,21 +333,26 @@
 	"Timer Button Zone Type - Both Buttons"
 	{
 		"en"		"Both buttons"
+		"chi"		"都为手动"
 	}
 	"Timer Button Zone Type - Only End Zone"
 	{
 		"en"		"Only end zone"
+		"chi"		"仅终点自动"
 	}
 	"Timer Button Zone Type - Both Zones"
 	{
-		"en"		"Both zones"
+		"en"		"Both Zones"
```

Comment:
Revert the english translation back to the original, this PR shouldn't touch them.